packages/components/app/styles/components/tag.scss

```diff
@@ -0,0 +1,92 @@
+//
+// TAG COMPONENT
+//
+// properties within each class are sorted alphabetically
+//
+//
+
+$hds-tag-border-radius: 13px;
+
+.hds-tag {
+  align-items: stretch;
+  background-color: var(--token-color-surface-interactive);
+  border: 1px solid var(--token-color-border-strong);
+  border-radius: $hds-tag-border-radius;
+  color: var(--token-color-foreground-primary);
+  display: inline-flex;
+  font-family: var(--token-typography-font-stack-text);
+  font-size: 0.8125rem; // 13px
+  font-weight: var(--token-typography-font-weight-medium);
+  line-height: 1rem; // 16px
+  vertical-align: middle;
+}
+
+.hds-tag__dismiss {
+  flex: 0 0 auto;
+  border-radius: inherit;
+  border-top-right-radius: 0;
+  border-bottom-right-radius: 0;
+  border: none; // reset default button border
+  margin: 0; // reset default button margin
+  padding: 6px 4px 6px 6px;
+}
+
+.hds-tag__dismiss-icon {
+  height: 12px;
+  width: 12px;
+}
+
+.hds-tag__text {
+  border-radius: inherit;
+  flex: 1 0 0;
+  padding: 4px 8px;
+}
+
+.hds-tag__link {
+  border-radius: inherit;
+  color: var(--token-color-foreground-action);
+  flex: 1 0 0;
+  padding: 4px 8px;
+
+  &:hover,
+  &.mock-hover {
+    color: var(--token-color-foreground-action-hover);
+  }
+
+  &:active,
+  &.mock-active{
+    color: var(--token-color-foreground-action-active);
+  }
+}
+
+.hds-tag__dismiss ~ .hds-tag__text,
+.hds-tag__dismiss ~ .hds-tag__link {
+  border-radius: inherit;
+  border-top-left-radius: 0;
```

Comment:
We do, otherwise we'll end up with something like this:
<img width="104" alt="Screenshot 2022-07-12 at 14 37 19" src="https://user-images.githubusercontent.com/788096/178503228-34cd4ce6-41fb-4f16-9517-a647c088a19b.png">

Reply:
Sorry, I removed the wrong line. I meant the line #64 (`border-radius: inherit;`)